spec/models/change_service_event_spec.rb

```diff
@@ -21,6 +21,19 @@
       expect(change_service_event.data['name']).to eql(new_service_name)
     end
   end
+  context '#trigger_publish_event' do
+    it 'when change service event occurs it triggers a publish event' do
+      component_1 = create(:sp_component)
```

Comment:
you don't need `component_1 = create(:sp_component)`
this `event = create(:assign_sp_component_to_service_event, service: service)` already has an sp _component

Reply:
I already pointed that out: https://github.com/alphagov/verify-self-service/pull/373#discussion_r392156578

Reply:
Ah.. didn't see 👍

Reply:
force pushing and is clicking the "Resolve" button too soon is to blame 😄